backend/src/test/kotlin/ZeebeBlacklistStateTest.java

```diff
@@ -57,6 +45,7 @@ public class ZeebeBlacklistStateTest {
 
   @Container
   public static ZeebeContainer zeebeContainer = new ZeebeContainer()
+      .withEnv("ZEEBE_BROKER_EXPERIMENTAL_ROCKSDB_DISABLEWAL", "false")
```

Comment:
:question: What happened when you not add this? The test failed? Interesting.

Reply:
Yes, otherwise the test could not read anything from rocksdb. As I wrote in the PR description, this was surprising to me as well.